gulpfile.js

```diff
@@ -235,15 +235,21 @@ gulp.task('run-unit-tests', function (cb) {
 });
 
 gulp.task('run-test-integration', function() {
-  return gulp.src('./tests/{common,integration}/**/*_test.js', { read: false }).pipe(mocha());
+  return gulp.src(
```

Comment:
I use `Ctrl + Alt + L` a lot in WebStorms to reformat automatically the code, so that I don't have to do it manually. Unfortunately it's not completely customisable and some cases like that can't be configured, so I like to keep complex indentation the way WebStorm does it so that I can forget formatting and focus on the code.
Maybe we should share the same WebStorm coding style configuration.